Move verification key origin paragraph from appendix to sec cons (#393)
ietf-rats-wg/eat: draft-ietf-rats-eat.md

of a modified detached claims because the algorithms in the COSE registry are assumed to be
of sufficient cryptographic strength.

## Verification Keys {#verfication-key-sc}

In all cases there must be some way that the verification key is itself verified or determined to be trustworthy.
The key identification itself is never enough.
This will always be by some out-of-band mechanism that is not described here.

> For example, the verifier may be configured with a root certificate or a master key by the verifier system administrator.

Reviewer: ```suggestion
For example, the verifier may be configured with a root certificate or a master key by a trusted entity such as the verifier system administrator.
```

Author: The word "trusted" is intentionally never used in the EAT document because to me it is too subjective. Trusted for what purpose and against what level of attack?

> Often an X.509 certificate or an endorsement carries more than just the verification key.

Reviewer: An X.509 certificate could be regarded as an Endorsement conceptual message.

> Often an X.509 certificate or an endorsement carries more than just the verification key.

Reviewer: ```suggestion
Often an Endorsement, such as an X.509 certificate, carries more than just the verification key.
```

Author: Prefer to leave it open since we have only a conceptual definition of endorsement.

> For example, an X.509 certificate might have key usage constraints and an endorsement might have reference values.

Reviewer: ```suggestion
For example, an X.509 certificate might have key usage constraints and an Endorsement might have Reference Values.
```

Reviewer 2: The suggestion to capitalize these terms is contrary to comments received and acted upon last summer (from Eliot I think). The change took effect in -15.

Author: Yes, we're using lower case.

> When this is the case, the key identifier must be either a protected header or in the payload such that it is cryptographically bound to the EAT.

Reviewer: ```suggestion
When this is the case, the key identifier must be either a protected header or in the payload, such that it is cryptographically bound to the EAT.
```
